data/core/terrain.cfg

```diff
@@ -128,6 +128,19 @@
     editor_group=water
 [/terrain_type]
 
+[terrain_type]
+    symbol_image=water/seaweed/kelp-tile
+    id=sea_kelp
+    name= _ "Swamp"
+    editor_name= _ "Kelp Forest"
```

Comment:
Shouldn't name and editor_name be the same?

Reply:
No, the editor names seem to be more descriptive.  I don't know that "Swamp" is necessarily the right thing here, but at least it is informative of the current situation.

Reply:
Okay, sure, maybe editor name is more descriptive, but… it doesn't seem correct for it to be completely different as it is in this case.